### PR TITLE
Fix #648 add aravis, epics-asyn, epics-pvxs

### LINKED ISSUE #647: Add args to beamsim_codes_install

### LINKED ISSUE #648: Add aravis, asyn, and pvxs
Needed for proprietary project so not in beamsim codes.

### PR DESCRIPTION
**Pull request by robnagler**
Fix #647 beamsim_codes_install takes list of codes as args 
- Added codes_epics_make_install for pvxs and asyn

### CODE REVIEW

File: installers/beamsim-codes/radiasoft-download.sh [hunk 1]
```diff
@@ -1,4 +1,18 @@
 #!/bin/bash
+#
+# Usage for install:
+#
+# radia_run beamsim-codes [codes...]
+#
+# With args, installs "everything" (see list below and _beamsim_codes_install_skip).
```
Comment (e-carlin) [inline installers/beamsim-codes/radiasoft-download.sh:7]: Without args

File: installers/rpm-code/codes/aravis.sh [hunk 2]
```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+aravis_main() {
+    # Not visible in GH tabs: https://github.com/AravisProject/aravis/releases
+    codes_yum_dependencies \
+        cmake \
```
Comment (e-carlin) [inline installers/rpm-code/codes/aravis.sh:6]: Doesn't this come from common? Many of our codes depend on cmake but don't re-list it as a dep.

File: installers/rpm-code/codes/aravis.sh [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+        meson \
+        python-gobject
+    codes_dependencies common
+    codes_download https://github.com/AravisProject/aravis/releases/download/0.8.31/aravis-0.8.31.tar.xz
+    meson setup \
+        --prefix="${codes_dir[prefix]}" \
+        --libdir="${codes_dir[prefix]}"/lib build
+    cd build
+    ninja -j"$(codes_num_cores)"
+    ninja install
```
Comment (e-carlin) [inline installers/rpm-code/codes/aravis.sh:29]: FWIW - I was curious about this. The meson docs linked to https://neugierig.org/software/chromium/notes/2011/02/ninja.html

I know some of our other code(s?) have the option to use ninja but I've turned it off.

**Review by e-carlin — APPROVED** [review]

File: installers/rpm-code/codes/aravis.sh (hunk 3 above)
Comment (robnagler) [reply installers/rpm-code/codes/aravis.sh:29]: neugierig.org got me curious. Went down a long rabbit hole. The number of people out there who are amazing is, well, amazing. Thanks.

I had no idea what ninja is or why. I just followed the instructions.

File: installers/rpm-code/codes/aravis.sh (hunk 2 above)
Comment (robnagler) [reply installers/rpm-code/codes/aravis.sh]: Removed in ba580fcd4a56104ca9c0e4ea760b812a3a6ef43e. I thought about distilling the deps, but this comes from the instructions, so I thought it was easier to just leave it there.

**Pull request merged**